docs/providers/aws/guide/layers.md

```diff
@@ -176,3 +176,23 @@ functions:
     layers:
       - { Ref: TestLambdaLayer }
 ```
+
+You can also configure layers at the service level. EG:
+
+```yml
+# serverless.yml
+service: aws-nodejs
+
+provider:
+  name: aws
+  runtime: nodejs8.10
```

Comment:
Let's not use an outdated runtime in the example

Reply:
Good call. Lazy copy-paste on my part